bot/ormconfig.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
```

Comment:
You had have already observed this, but wanted to place it as a reminder here - if this file is no longer needed it can be deleted.

Reply:
Thanks ya I deleted it.